cysync/src/pages/mainApp/sidebar/index.tsx

```diff
@@ -184,6 +185,10 @@ const TabValues = [
     tab: 2,
     route: Routes.wallet.index
   },
+  {
+    tab: 10, // this is a dummy value for the Swap tab
+    route: Routes.swap.index //TODO: Update the route
+  },
```

Comment:
Add it at the 2nd last index of array and remove comments

Reply:
Also update the values of tab index

Reply:
The values of just this particular tab or the rest of the tabs as well?